### ISSUE TITLE
Cannot change icon for star

### ISSUE THREAD

**Issue by mangofeet** [ben/foundry-ironsworn#581]
When I create a new star, the "mystery man" icon shows up and for some reason no matter what other file I select (on the sheet *and* on the token config) the token image for the star does not change. 

I tried this on a fresh world created with 1.20.8, also on a completely fresh foundry data directory and it was still happening (haven't tried totally reinstalling foundry itself, I upgraded my instance from the built in tool yesterday)

**Comment by ben** [comment]
Odd. This totally works for me. It's a bit trickier for stars than it is for planets (when you choose the planet type, it sets the actor and token images for you), but this flow works:

![CleanShot 2023-01-01 at 06 43 17](https://user-images.githubusercontent.com/39902/210174647-85223002-9a3a-422d-90bc-ec2460e0a084.jpg)

Did you end up in the "Prototype token" settings? If so, you can drag a new copy of your actor onto the map and it'll have the new token image.

**Comment by mangofeet** [comment]
I changed the icon as in your picture and it still didn't change for me, strange..

**Comment by ulilaube** [comment]
Hi! I have a similar effect in FVTT 10.291 setup up with Ironsworn & Starforged 1.20.28. If I include one of the svg assets in a journal page as image, e.g. https://github.com/ben/foundry-ironsworn/blob/main/system/assets/icons/stellar-object.svg then it is rendered as expected, as it is in the preview in the file-chooser-dialog. If I create a token with that svg file as an image I only get the default hooded head. That is with Firefox 110.1 and Google Chrome 110.0.5481.178.

If I take a svg from the Pathfinder2e game system e.g. https://github.com/foundryvtt/pf2e/blob/master/static/assets/icons/chevron-down.svg the token shows the chevron as expected.

Both SVG files use different SVG-primitives: path vs. polyline. Does the canvas rendering subsystem of FVTT support all the various SVG primitives?

When I press F12 in Firefox and go to the JavaScript-Konsole and then "shift reload" the whole scene with the token in question, then I get:

Error: Invalid BaseTexture systems/foundry-ironsworn/assets/icons/stellar-object.svg
    loadTexture foundry.js:26082
    _draw foundry.js:46145
    draw foundry.js:25199
    promises foundry.js:30825
    map commons.js:3332
    _draw foundry.js:30823
    _draw foundry.js:37066
    draw foundry.js:30349
    draw foundry.js:26151
    draw foundry.js:39270
    draw foundry.js:23673
    view foundry.js:20901
    initializeCanvas foundry.js:8095
    setupGame foundry.js:7929
    _initializeGameView foundry.js:8999
    _initializeView foundry.js:8975
    initialize foundry.js:7825
    call_wrapped libWrapper-wrapper.js:507
    libWrapperInit libWrapper-api.js:803
    initialize#0 libWrapper-wrapper.js:187
    <anonymous> foundry.js:79693
    async* foundry.js:79675

**Comment by ulilaube** [comment]
That happens with other svg files as well.

Error: Invalid BaseTexture systems/foundry-ironsworn/assets/icons/region-sf.svg
    loadTexture https://ulsd.uber.space/scripts/foundry.js:26082
    _draw https://ulsd.uber.space/scripts/foundry.js:46145
    draw https://ulsd.uber.space/scripts/foundry.js:25199
    _onUpdateAppearance https://ulsd.uber.space/scripts/foundry.js:47186
    _onUpdate https://ulsd.uber.space/scripts/foundry.js:47102
    _onUpdate https://ulsd.uber.space/scripts/foundry.js:12367
    _onUpdate https://ulsd.uber.space/scripts/foundry.js:22373
    _previewChanges https://ulsd.uber.space/scripts/foundry.js:71120
    _onChangeInput https://ulsd.uber.space/scripts/foundry.js:71103
    jQuery 2
    _onSubmit https://ulsd.uber.space/scripts/foundry.js:58555
    onsubmit https://ulsd.uber.space/scripts/foundry.js:58310
    activateListeners https://ulsd.uber.space/scripts/foundry.js:58310
    _render https://ulsd.uber.space/scripts/foundry.js:5138
    render https://ulsd.uber.space/scripts/foundry.js:5067
    render https://ulsd.uber.space/scripts/foundry.js:58268
    _onChangeDisplayMode https://ulsd.uber.space/scripts/foundry.js:58333
    jQuery 8
    activateListeners https://ulsd.uber.space/scripts/foundry.js:58294
    _render https://ulsd.uber.space/scripts/foundry.js:5138
    render https://ulsd.uber.space/scripts/foundry.js:5067
    render https://ulsd.uber.space/scripts/foundry.js:58268
    browse https://ulsd.uber.space/scripts/foundry.js:58126
    _onPick https://ulsd.uber.space/scripts/foundry.js:58454
    jQuery 8
    activateListeners https://ulsd.uber.space/scripts/foundry.js:58303
    _render https://ulsd.uber.space/scripts/foundry.js:5138
    render https://ulsd.uber.space/scripts/foundry.js:5067
    render https://ulsd.uber.space/scripts/foundry.js:58268
    browse https://ulsd.uber.space/scripts/foundry.js:58126
[foundry.js:26089:13](https://ulsd.uber.space/scripts/foundry.js)
    loadTexture https://ulsd.uber.space/scripts/foundry.js:26089
    _draw https://ulsd.uber.space/scripts/foundry.js:46145
    draw https://ulsd.uber.space/scripts/foundry.js:25199
    _onUpdateAppearance https://ulsd.uber.space/scripts/foundry.js:47186
    _onUpdate https://ulsd.uber.space/scripts/foundry.js:47102
    _onUpdate https://ulsd.uber.space/scripts/foundry.js:12367
    _onUpdate https://ulsd.uber.space/scripts/foundry.js:22373
    _previewChanges https://ulsd.uber.space/scripts/foundry.js:71120
    _onChangeInput https://ulsd.uber.space/scripts/foundry.js:71103
    jQuery 2
    _onSubmit https://ulsd.uber.space/scripts/foundry.js:58555
    onsubmit https://ulsd.uber.space/scripts/foundry.js:58310
    (Async: EventHandlerNonNull)
    activateListeners https://ulsd.uber.space/scripts/foundry.js:58310
    _render https://ulsd.uber.space/scripts/foundry.js:5138
    render https://ulsd.uber.space/scripts/foundry.js:5067
    render https://ulsd.uber.space/scripts/foundry.js:58268
    _onChangeDisplayMode https://ulsd.uber.space/scripts/foundry.js:58333
    jQuery 8
    activateListeners https://ulsd.uber.space/scripts/foundry.js:58294
    _render https://ulsd.uber.space/scripts/foundry.js:5138
    render https://ulsd.uber.space/scripts/foundry.js:5067
    render https://ulsd.uber.space/scripts/foundry.js:58268
    browse https://ulsd.uber.space/scripts/foundry.js:58126
    _onPick https://ulsd.uber.space/scripts/foundry.js:58454
    jQuery 8
    activateListeners https://ulsd.uber.space/scripts/foundry.js:58303
    _render https://ulsd.uber.space/scripts/foundry.js:5138
    render https://ulsd.uber.space/scripts/foundry.js:5067
    render https://ulsd.uber.space/scripts/foundry.js:58268
    browse https://ulsd.uber.space/scripts/foundry.js:58126

**Comment by ulilaube** [comment]
If I load it in PixiJS 6.5.2 in Firefox 110.1 directly

```
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="Content-Security-Policy" content="default-src * 'unsafe-inline';">
	<script src="pixi.min.js"></script>
	<script src="unsafe-eval.min.js"></script>
  </head>
  <body>
    <script>
      let app = new PIXI.Application({ width: 640, height: 360 });
	  document.body.appendChild(app.view);
	  const texture = PIXI.Texture.from('stellar-object.svg');
	  const sprite = new PIXI.Sprite(texture);
	  app.stage.addChild(sprite);
    </script>	
  </body>
</html>
```

I get 

`Uncaught Error: The SVG image must have width and height defined (in pixels), canvas API needs them.`

Looking at the SVG file and changing it from

`<svg style="height: 512px; width: 512px;" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">`

to

`<svg height="512" width="512" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">`

removes the error and displays the stellar object symbol. Google Chrome doesn't care and shows the symbol in both cases, without the error msg.

**Comment by ulilaube** [comment]
If I change all the svg files from the asset/icons folder like that, I do not get the hooded head default symbol in any case any more. That fixes it for me.

**Comment by ben** [comment]
Interesting! And that doesn't break the other places where those icons are used, like the sector tools sidebar?

**Comment by ulilaube** [comment]
I wasn't precise enough. There are .svg files in assets/icons that have 512x512 in them. Some like to logo ones seem to be 32x32. Ihave only changed the 512x512 ones as shown above.

The sector tools toolbar seems no to be affected by this change. It properly aligned, the size of the icons is uniform.

![grafik](https://user-images.githubusercontent.com/1592734/222969819-a5404a8f-790a-4989-8ddc-406369738c2d.png)

**Comment by ulilaube** [comment]
Other assets seem to affected too

![grafik](https://user-images.githubusercontent.com/1592734/222970798-34d2b40f-73ab-4958-b779-e0e2b5dd0d7c.png)

The 'Vault Planetside.svg'  svg has no  width  or height attributed at all. neitehr explicit nor as styles-string. If I change that. by adding  height="128" width="128". It fixes that as well.

![grafik](https://user-images.githubusercontent.com/1592734/222971139-dfe14447-ec22-463d-8710-89d812af71c0.png)

There is this post in the FVTT Discord troubleshooting chat

https://discord.com/channels/170995199584108546/486930822465716249/1070414342358573166

**Comment by ulilaube** [comment]
oh oh that seems to be nothing new with Firefox ...

https://bugzilla.mozilla.org/show_bug.cgi?id=779368
https://bugzilla.mozilla.org/show_bug.cgi?id=736431

**Comment by ben** [comment]
This is great work. Want to submit a pull request?

**Comment by ben** [comment]
Should be fixed with the merging of #663. Thanks!
